packages/rest/test/unit/request-body.validator.test.ts

```diff
@@ -333,9 +333,6 @@ function verifyValidationRejectsInputWithError(
 ) {
   try {
     validateRequestBody(body, spec, schemas);
-    throw new Error(
```

Comment:
Any reason to remove this line? 
Without it if `validateRequestBody(body, spec, schemas)` doesn't throw error, the test will pass while it's expected to fail.

Reply:
That `Error` was to  "emit" a wrong error if the `try`-`catch` in previous code did not throw. We are not `try`-`catching` anymore.

Reply:
> We are not try-catching anymore.

This does not match my understanding of this test code, I still see try/catch in place.

Please keep revert the changes in this test.

Reply:
Updating to be consistent.